src/main/flight/imu.c

```diff
@@ -476,7 +640,10 @@ STATIC_UNIT_TESTED float imuCalcCourseErr(float courseOverGround)
     // for error angles less than 90 degrees (dot > 0), use cross value to compute ez_ef
     //   over 90 deg, use sign(cross)
     // when well aligned, return ~ 0 (sin(error)), for error >= 90 degrees, return +-1.0
-    return (dot > 0) ? cross : (cross < 0 ? -1.0f : 1.0f);
+    const float scalarErr = ((dot > 0) ? cross : (cross < 0 ? -1.0f : 1.0f) );
```

Comment:
(technically, this is error vector with Z component only [cross product of vectors in xy plane], not scalar)